code/UI/General/TabMenus/Scoreboard/Entry/ScoreboardEntry.cs

```diff
@@ -45,4 +77,28 @@ public void Update()
 
 		PlayerAvatar.SetTexture( $"avatar:{_client.PlayerId}" );
 	}
+
+	public void OnClick()
+	{
+		// Only allow tagging currently alive players, and only if the selected player is not the local player
+		if ((_client.Pawn as Player).Status is not PlayerStatus.Alive || _client.Pawn.IsLocalPawn)
+			return;
+
+		if (!Expanded) {
+			Style.Set("height", "76px");
+			GetChild(1).RemoveClass("no-display");
```

Comment:
We have a `Enabled(...)` util that you can use to enable/disable UI elements. Let's use that here.

Instead of using `GetChild(1)`, let's just grab a reference to to the div instead. Exactly what you did with `Tag`, except it's a panel.